packages/vite-plugin-web-extension/src/options.ts

```diff
@@ -62,7 +62,7 @@ export interface UserOptions {
    *
    * @default "chrome"
    */
-  browser?: string;
+  browser?: 'chrome' | 'firefox' | null;
```

Comment:
This should remain a `string`. You can target more than just chrome or firefox. Safari, opera, etc.

Reply:
Oops! That's strictly for the manifest templating? I don't see any handling for Safari and others in `createWebExtRunner`, hence the assumption. 😄